src/runtime/Loader.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
+
+interface Props {
+  render: JSX.Element | undefined;
+}
+export const LoaderContext = React.createContext(undefined);
+
+export class Loader extends Component<Props> {
+  public state = {
```

Comment:
please follow the patterns present in the rest of the code: https://github.com/dOrgTech/DAOcomponents/blob/88331d47d338e9aee1defdb1bebc5ec9c04138fe/src/runtime/Component.tsx#L57